src/Factory.sol

```diff
@@ -247,23 +269,23 @@ contract Factory {
                 timelock,
                 abi.encodeCall(
                     WithdrawalQueue.initialize,
-                    (timelock, vaultConfig.nodeOperator) // (admin, finalizerRoleHolder)
+                    (timelock, _vaultConfig.nodeOperator) // (admin, finalizerRoleHolder)
                 )
             )
         );
 
-        address distributor = DISTRIBUTOR_FACTORY.deploy(vaultConfig.nodeOperator, vaultConfig.nodeOperatorManager);
+        address distributor = DISTRIBUTOR_FACTORY.deploy(_vaultConfig.nodeOperator, _vaultConfig.nodeOperatorManager);
```

Comment:
Distributor is deployed not under a proxy, so it's better to have an opportunity to recover eth from it. consider adding a recover method